content/_index.md

```diff
@@ -3,50 +3,4 @@ title: "Open Service Mesh"
 description: "Open Service Mesh documentation and resources."
```

Comment:
Will this be the landing page for docs.openservicemesh.io?

Reply:
@jaellio yes. this content was difficult to return to from the docs site itself. i moved it to an about page to make it more accessible.